in/index.html.mako

```diff
@@ -100,6 +100,8 @@
           <th class="storage">Storage</th>
           <th class="architecture">Arch</th>
           <th class="ioperf">I/O Performance</th>
```

Comment:
Probably makes sense to call this column "Network Perf" now that it is just representing that field?